### ISSUE TITLE
Is the dataset private?

### ISSUE THREAD

**Issue by SachidanandAlle** [navidstuv/NuClick#23]
How I can access the dataset without sign-in?  Is it private?
Is there a public version of this dataset available?

Also looking for pytorch based implementation.. :)

**Comment by SachidanandAlle** [comment]
The idea is to have this e2e as a monai label app for users to try and experiment that includes both infer and finetune a model..
https://github.com/Project-MONAI/MONAILabel/
